hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/SecurityManagedConfiguration.java

```diff
@@ -609,13 +609,20 @@ CorsConfigurationSource corsConfigurationSource() {
         private final VaadinUrlAuthenticationSuccessHandler handler;
 
         @Autowired(required = false)
-        private AuthenticationSuccessHandler oidcAuthenticationSuccessHandler;
+        private OAuth2UserService<OidcUserRequest, OidcUser> oidcUserService;
 
         @Autowired(required = false)
-        private LogoutHandler oidcLogoutHandler;
+        private AuthenticationEntryPoint authenticationEntryPoint;
```

Comment:
Check if AuthenticationEntryPoint has to be a bean or can be added as new or inline var.

Reply:
done.